MSAL/src/instance/oauth2/MSALOauth2Provider+Internal.h

```diff
@@ -25,14 +25,20 @@
 //
 //------------------------------------------------------------------------------
 
-#import <Foundation/Foundation.h>
+#import "MSALOauth2Provider.h"
 
 @class MSIDOauth2Factory;
 
-@interface MSALOauth2FactoryProducer : NSObject
+#ifndef MSALOauth2BaseFactory_Internal_h
+#define MSALOauth2BaseFactory_Internal_h
 
-+ (nullable MSIDOauth2Factory *)msidOauth2FactoryForAuthority:(nonnull NSURL *)authority
-                                                      context:(nullable id<MSIDRequestContext>)context
-                                                        error:(NSError * _Nullable __autoreleasing * _Nullable)error;
+@interface MSALOauth2Provider()
+
+@property (nonatomic, nonnull, readwrite) MSIDOauth2Factory *msidOauth2Factory;
+
+- (void)initDerivedProperties;
```

Comment:
nit: should we be more specific? and does this need to be public? i.e/ would other class call this method directly?

Reply:
Fixed